Alondra Miranda Aguilera A01746742 Grupo 2

A01746742 opened the pull request.

A01746742 committed:
Alondra Miranda Aguilera A01746742 Grupo 2

A01746742 committed:
Alondra Miranda Aguilera A01746742 Grupo 2

Este es un programa para enseñar arte contemporáneo. Primero te da la definición de arte, según tu personalidad. Después te muestra información y características de 3 movimientos importantes que son Expresionismo abstracto, Pop Art y Minimalismo. Al final tienes la opción de hacer un quizz de 3 preguntas.

rmroman commented:
Tu proyecto marca error, puedes subir correcciones. Si no lo haces, así califico tu proyecto.

    5. Quizz
    0. Salir
    
Elige una opcción: 5
¿Cuántas de esas obras pertenecen al minimalismo? 1, 2 o 3: 1
¿Cuántas de esas obras pertenecen al popart? 1, 2 o 3: 2
Traceback (most recent call last):
  File "ProyectoFinal.py", line 199, in <module>
    main()
  File "ProyectoFinal.py", line 192, in main
    quizz()
  File "ProyectoFinal.py", line 149, in quizz
    puntaje = puntaje + puntos[2]
IndexError: list index out of range


------------------
(program exited with code: 1)
Press return to continue

rmroman commented:
.

A01746742 committed:
Alondra Miranda Aguilera | A01746742 | Grupo 2

Profe, añado el programa ya corregido y un video probando que en mi compu me corre super bien el Quizz y cuando le doy salir :D 
Muchas gracias por darme la oportunidad de corregir.

A01746742 commented:
Profe gracias por darme la oportunidad para corregir el programa. Ya se lo puse aquí, se llama "ProyectoFinalELBUENO" y puse igual un video de pantalla donde en mi compu si me jala el quizz y salir.

rmroman commented:
# Retroalimentación.
## Proyecto final


Rubro | Puntos | Observaciones
-- | -- | --
Gráficos | 200/200 | Muy bien
Listas | 200/200 | Muy bien
Elementos en pantalla (>=2 objetos) | 200/200 | Muy bien
Interacción  | 200/200 | Muy bien
Menú  | 200/200 | Muy bien
Acerca de  | 0/200 | Muy bien

Has ganado **1000/1200 HP**.

¡Excelente!
💯
